docs/src/pages/components/tree-view/GmailTreeView.tsx

```diff
@@ -50,6 +50,10 @@ const useTreeItemStyles = makeStyles((theme: Theme) =>
         backgroundColor: `var(--tree-view-bg-color, ${theme.palette.action.selected})`,
         color: 'var(--tree-view-color)',
       },
+      '& $label': {
```

Comment:
To reduce the effort once we migrate the docs?

```suggestion
      '& .MuiTreeItem-label': {
```

Reply:
let's keep it and fix all again (there are other places that use `$` also) when we migrate the docs